website/docs/docs/remote-cache/github-actions/configuration.md

```diff
@@ -13,6 +13,54 @@ Make sure to include the following workflow permissions for your project:
 
 Settings -> Actions -> General -> Workflow Permissions -> **Read and write permissions**
 
+## Optimizing CI/CD Performance with paths-ignore
+
+When using GitHub Actions workflows with RNEF, you can optimize your CI/CD pipelines by using `paths-ignore` to skip unnecessary workflow runs. This can significantly reduce CI time and costs, especially in large repositories where not all changes require rebuilding the mobile applications.
+
+### Why use paths-ignore?
+
+- **Save CI/CD minutes**: Skip mobile app builds when only irrelevant files change
+- **Faster feedback cycles**: Only run workflows when truly necessary
+- **Cost optimization**: Reduce computational resources needed for CI/CD
+
+### How to implement paths-ignore
+
+Add a `paths-ignore` section to your workflow's trigger configuration to specify which file patterns should not trigger the workflow:
+
+```yaml
+name: Mobile Build
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '*.md'                    # Skip documentation changes
+      - 'docs/**'                 # Skip documentation directory
+      - '.github/ISSUE_TEMPLATE/**' # Skip issue templates
+      - 'web/**'                  # Skip web-specific code
+      - 'server/**'               # Skip backend code
+      - 'design/**'               # Skip design files
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - '*.md'
+      - 'docs/**'
+      - '.github/ISSUE_TEMPLATE/**'
+      - 'web/**'
+      - 'server/**'
+      - 'design/**'
```

Comment:
we can convert it to a comment: `# you can set similar config for pull_request hook`